style.css

```diff
@@ -0,0 +1,454 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
+
+* {
+  font-family: 'Inter', sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] Kindly make sure your page fits size of the screen. The user does not need to scroll to the right to see the entirety of the page.
![image](https://user-images.githubusercontent.com/97100091/213299939-918869b1-f8ca-4b37-b03e-2f41bfe0684e.png)